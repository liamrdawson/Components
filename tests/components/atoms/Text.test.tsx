import { jsx } from '@emotion/react'
import {render} from '@testing-library/react'
import { Text } from '../../../src'

test('Snapshot Test: Text', () => {
  const {container} = render(
    <Text element="a" fontWeight="light" fontSize="medium">
      I am a string
    </Text>,
  )
  expect(container).toMatchSnapshot()
})

test('Snapshot Test: Text', () => {
  const {container} = render(
    <Text element="p" fontWeight="medium" fontSize="large">
      I am a string
    </Text>,
  )
  expect(container).toMatchSnapshot()
})
