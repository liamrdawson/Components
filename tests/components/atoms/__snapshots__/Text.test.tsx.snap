// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test: Text 1`] = `
<div>
  <a
    class="css-zkrtb7-Text"
  >
    I am a string
  </a>
</div>
`;

exports[`Snapshot Test: Text 2`] = `
<div>
  <p
    class="css-1aktc3o-Text"
  >
    I am a string
  </p>
</div>
`;
